src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_EditProfileScreen.tsx

```diff
@@ -0,0 +1,324 @@
+                  mode="primary"
+                  text={t(EditProfileTexts.button.save)}
+                  onPress={onSubmit}
+                  style={styles.submit}
+                  disabled={isLoadingOrSubmittingProfile}
+                  rightIcon={
+                    submissionStatus === 'loading'
+                      ? {
+                          svg: ArrowRight,
+                          loading: submissionStatus === 'loading',
+                        }
+                      : undefined
+                  }
+                />
+                {submissionStatus === 'submitted' && (
+                  <MessageBox
+                    type="valid"
+                    message={t(EditProfileTexts.profileUpdate.success)}
+                  />
+                )}
+                {submissionStatus === 'submission-error' && (
+                  <MessageBox
+                    type="error"
+                    message={t(EditProfileTexts.profileUpdate.error)}
+                  />
+                )}
+              </Section>
+              <View style={styles.profileContainer}>
+                <ThemeText style={styles.profileText} color="secondary">
+                  {t(EditProfileTexts.profileInfo.profile)}
+                </ThemeText>
+                <ThemeText>
+                  {t(EditProfileTexts.profileInfo.loginProvider)}
+                </ThemeText>
+                <ThemeText
+                  type="body__secondary"
+                  color="secondary"
+                  style={styles.profilePhone}
+                >
+                  {t(EditProfileTexts.profileInfo.otp(phoneNumber))}
+                </ThemeText>
+                {customerNumber && (
+                  <>
+                    <ThemeText>
+                      {t(EditProfileTexts.profileInfo.customerNumber)}
+                    </ThemeText>
+                    <ThemeText
+                      type="body__secondary"
+                      color="secondary"
+                      accessibilityLabel={numberToAccessibilityString(
+                        customerNumber,
+                      )}
+                    >
+                      {customerNumber}
+                    </ThemeText>
+                  </>
+                )}
+              </View>
+            </>
+          )}
+          <Section withPadding withTopPadding withBottomPadding>
+            <Button
+              mode="primary"
+              interactiveColor="interactive_destructive"
+              leftIcon={{svg: Delete}}
+              text={t(EditProfileTexts.button.deleteProfile)}
+              onPress={() => navigation.navigate('Profile_DeleteProfileScreen')}
+            />
+          </Section>
+        </>
+      )}
+    </FullScreenView>
+  );
+};
+
+function isValidEmail(email: string) {
+  // Just a really simple check to avoid the most obvious wrong ones.
+  // More validation is done on the server
+  return /^\S+@\S+\.\S+$/.test(email);
+}
```

Comment:
I guess this is relevant here: https://mittatb.slack.com/archives/C0116FMPX4Y/p1700468560795789

We have an util function, though it isn't perfect.

Reply:
Relevant indeed. Should probably use [the existing util function](https://github.com/AtB-AS/mittatb-app/blob/master/src/utils/validation.ts) then.
If it has issues, it could be worth checking whether those are solved using the suggestion from StackOverflow above (it has quite a lot of upvotes).

Reply:
This was a raw copy of the email validation we have in Webshop for updating profile. 
Ref. https://github.com/AtB-AS/webshop2/blob/793c5b0ca114454961f70f5846f399d3570a4637/src/pages/profile/profile-info.tsx#L152 
Used it here as we stated in the issue (https://github.com/AtB-AS/kundevendt/issues/4275) to use the same input validation in edit profile screen in the app as in the existing one in webshop. 
Do you still think we should change it to what you suggest @marius-at-atb ? If so, what about webshop? :)

Reply:
Have tested with some weird, but valid, addresses - and both the util function and the Stackoverflow seems to handle them. However, even though there are few (if any?) top level domains ending with a digit, I guess it's actually allowed as I interpret [RFC 3696](https://datatracker.ietf.org/doc/html/rfc3696#section-2):

```
a domain name must consist of only the ASCII alphabetic and numeric characters, plus the hyphen....
There is an additional rule that essentially requires that top-level domain names not be all-numeric.
```

Either way, I guess both works for all practical reasons. 

When it comes to testing, we could add a unit test for this case. There is also a possibility to add component tests through the e2e-framework Cypress to test the email input field in isolation. Will look into such things a bit later.

Reply:
Concluded with using the same validation as Entur, so swapped it with their Regex 📧